packages/terra-list/src/SingleSelectList.jsx

```diff
@@ -129,7 +136,11 @@ class SingleSelectList extends React.Component {
       newProps.onKeyDown = onKeyDown;
     }
 
+    // Uses the props.hasChevron value, unless the consumer specifies the item hasChevron attribute as false.
     newProps.hasChevron = this.props.hasChevrons;
+    if (item.props.hasChevron === false) {
```

Comment:
The default prop for `hasChevrons` is false, so a Single Select List will always give a list item a `hasChevron` value of false. When a Single Select List is passed hasChevrons, it will give every list item a `hasChevron` value of true, unless a specific item was given `hasChevron={false}`. I am not sure undefined needs checked. Can you think of an instance where it would?